### PR TITLE
Added Crypress tests

### PR DESCRIPTION
**Pull request by michaelk566**
Added extensive cypress tests for tutorials which include testing over various languages. Also added many tests for a user trying multiple combinations of different settings options.

### CODE REVIEW

File: cypress/integration/main/combination.spec.js [hunk 1]
```diff
@@ -0,0 +1,622 @@
+const background_1_path = 'url("http://127.0.0.1:5500/images/background.svg")';
+const background_2_path = 'url("http://127.0.0.1:5500/images/background2.svg")';
+const background_3_path = 'url("http://127.0.0.1:5500/images/background3.svg")';
+
+
+describe('Combinations of background and colorblind', () => {
```
Comment (jinbest17) [inline cypress/integration/main/combination.spec.js:6]: good job on testing multiple features in integration.

File: cypress/integration/main/languages.specs.js [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
-    cy.get("#stats-header").should('have.text', 'Estadísticas de usuario');
+    cy.get('#pomobear-header').should('have.text', 'Oso pomodoro');
+    cy.get('#settings-header').should('have.text', 'Configuración');
+    cy.get('#stats-header').should('have.text', 'Estadísticas de usuario');
   });
 
-//   it("Changing language multiple times ", () => {
-//     cy.get('select').eq(1).select('enUS');
-//     cy.get("#languageDropDownText").should('have.text', 'Change Language');
+  it('Changing language multiple times and repeat', () => {
```
Comment (jinbest17) [inline cypress/integration/main/languages.specs.js:31]: good job on testing repetitive user actions.

**Review by jinbest17 — APPROVED** [review]
Good job on adding e2e cypress tests with combined user action.

File: cypress/integration/main/languages.specs.js (hunk 2 above)
Comment (karinasanchez04) [reply cypress/integration/main/languages.specs.js:31]: Good job on thoroughly testing different combinations of different user actions!

**Review by karinasanchez04 — APPROVED** [review]
Good job overall on adding testing tests that are thorough for the features we have.

**Pull request merged**